example/tsconfig.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
Why is this change necessary?

Reply:
This it’s automatically generated by expo when you start the app, I believe it finds for a tsconfig and if there’s none, it creates this for you.